pkg/cmd/server/admin/create_mastercerts.go

```diff
@@ -15,6 +16,49 @@ import (
 )
 
 const CreateMasterCertsCommandName = "create-master-certs"
+const master_cert_long = `
+This command creates keys and certs necessary to run a secure OpenShift master.
+It also creates keys, certificates, and configuration necessary for most
+related infrastructure components that are clients to the master.
+See the related "create-node-config" command for generating per-node config.
+
+All files are expected or created in standard locations under the cert-dir.
+
+    openshift.local.config/master/
+	    ca.{crt,key,serial.txt}
+	    master.server.{crt,key}
+		openshift-router.{crt,key,kubeconfig}
+		admin.{crt,key,kubeconfig}
+		...
+
+Note that the certificate authority (CA aka "signer") generated automatically
+is self-signed. In production usage, administrators are more likely to
+want to generate signed certificates separately rather than rely on an
+OpenShift-generated CA. Alternatively, start with an existing signed CA and
+have this command use it to generate valid certificates.
+
+This command would usually only be used once at installation. If you
+need to regenerate the master server cert, DO NOT use --overwrite as this
+would recreate ALL certs including the CA cert, invalidating any existing
+infrastructure or client configuration. Instead, delete/rename the existing
+server cert and run the command to fill it in:
+
+    $ mv openshift.local.config/master/master.server.crt{,.old}
+    $ %[1]s --cert-dir=... \
+	        --master=https://internal.master.fqdn:8443 \
+            --public-master=https://external.master.fqdn:8443 \
+            --hostnames=external.master.fqdn,internal.master.fqdn,localhost,127.0.0.1,172.17.42.1,kubernetes.default.local
+
+Alternatively, use the related "create-server-cert" command to explicitly
+create a certificate.
+
+Regardless of --overwrite, the master server key/cert will be updated 
```

Comment:
@liggitt Thoughts on this?  Read all the way through.  We sometimes want it and sometimes don't...